setup.py

```diff
@@ -1,37 +1,35 @@
 from setuptools import setup
 
 setup(
-    name='klangbecken',
-    version='0.0.12',
-    description='Klangbecken Audio Player',
-    url='https://github.com/radiorabe/klangbecken',
-    author='Marco Schmalz',
-    author_email='marco@schess.ch ',
-    py_modules=['klangbecken_api', 'saemubox_listener'],
-    license='AGPLv3',
+    name="klangbecken",
+    version="0.0.12",
+    description="Klangbecken Audio Player",
+    url="https://github.com/radiorabe/klangbecken",
+    author="Marco Schmalz",
+    author_email="marco@schess.ch ",
+    py_modules=["klangbecken_api", "saemubox_listener"],
+    license="AGPLv3",
     classifiers=[
-        'Development Status :: 3 - Alpha',
-        'Environment :: Web Environment',
-        'Intended Audience :: Developers',
-        'License :: OSI Approved :: GNU Affero General Public License v3',
-        'Operating System :: POSIX',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6',
-        'Programming Language :: Python :: 3.7',
-        'Programming Language :: Python :: 3.8',
-        'Topic :: Internet :: WWW/HTTP :: WSGI',
+        "Development Status :: 3 - Alpha",
+        "Environment :: Web Environment",
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: GNU Affero General Public License v3",
+        "Operating System :: POSIX",
+        "Programming Language :: Python :: 2",
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Topic :: Internet :: WWW/HTTP :: WSGI",
     ],
-    install_requires=['docopt', 'mutagen', 'six', 'Werkzeug'],
-    extras_require={
-        'test': ['tox', 'coverage', 'mock', 'flake8'],
-    },
+    install_requires=["docopt", "mutagen", "six", "Werkzeug"],
+    extras_require={"test": ["tox", "coverage", "mock", "flake8"]},
```

Comment:
Should match lines in `requirements-dev.txt`

Reply:
Replaced with some black magic (ie reading from reqs in setup.py). Since we do not have plans on large scale distribution of this through pypi this is IMO fine for now.